site/content/docs/user/ingress.md

```diff
@@ -116,7 +116,8 @@ Additional information about Contour can be found at: [projectcontour.io](https:
 ### Ingress NGINX
 
 {{< codeFromInline lang="bash" >}}
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+VERSION=$(curl https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/stable.txt)
```

Comment:
one little nit: this now only works on mac/linux, this doc did work on windows too
when v1 is released we can revisit this perhaps

Reply:
Completely agreed. Will check with @cpanato the best way actually to have like a "stable" branch and always merge there

Reply:
maybe a copy of the stable version's yaml somewhere? 🤔

Reply:
hum, will think. Does sig-release have some storage bucket? Or maybe I can ask one for k8s.io workgroup

Reply:
k8s.io wg is the best place to we setup this, maybe we can setup a staging account, not sure